CHANGELOG.md

```diff
@@ -9,6 +9,32 @@
   Microcosm classes.
 - Fixed a bug where the first action dispatched would always fire a
   change event, even if state didn't change.
+- Action status changing methods are auto-bound, and will warn when a
+  completed action attempts to move into a new state (strict mode only)
+
+### Autobound action status methods
+
+Action status methods, like `action.resolve()`, or `action.reject()`
+are now auto-bound. This means that you can pass them directly into a
```

Comment:
feels like we should avoid phrases like `are now auto-bound` in documentation, and just say `are auto-bound`